react-native-reanimated.d.ts

```diff
@@ -484,10 +491,11 @@ declare module 'react-native-reanimated' {
     export type AnimatedStyleProp<T extends object> = AnimateStyle<T> | RegisteredStyle<AnimateStyle<T>>;
     export function useAnimatedStyle<
       T extends AnimatedStyleProp<ViewStyle | ImageStyle | TextStyle>
-    >(updater: () => T, deps?: DependencyList): T;
+    >(updater: () => T, deps?: DependencyList | null): T;
     export function useAnimatedProps<T extends {}>(
       updater: () => T,
-      deps?: DependencyList
+      deps?: DependencyList | null,
+      adapters?: PropsAdapterFunctionType | Array<PropsAdapterFunctionType> | null
```

Comment:
Please use `PropsAdapterFunctionType[]` instead of `Array<PropsAdapterFunctionType>`

Reply:
done a44f3d0